code/go/pkg/validator/validator_test.go

```diff
@@ -203,7 +203,14 @@ func TestValidateFile(t *testing.T) {
 			"manifest.yml",
 			[]string{
 				`field policy_templates.0.deployment_modes: Additional property default is not allowed`,
-				`field policy_templates.0.inputs.0.vars.0.hide_in_deployment_modes.0: policy_templates.0.inputs.0.vars.0.hide_in_deployment_modes.0 must be one of the following: "agentless"`,
```

Comment:
No need to remove this line, right?

Reply:
Aaah 💩 ,   I originally changed the wrong test.